### DISCUSSION TITLE
On going error with latest version of npm

### DISCUSSION THREAD

**Discussion by Dvalin21** [tteck/Proxmox#2068] [General]
This was more of a heads up. The script installs the latest version of Npm to a lxc, but there has been this issue
https://github.com/NginxProxyManager/nginx-proxy-manager/issues/2442

**Comment by DomJeal** [comment]
I used the script to install an NPM LXC the other day and I'm able to add certificates, not had to renew any yet.

Are you able to add?

**Reply by Dvalin21** [reply]
Neither add or renew. However, I'm having to basically start from scratch due to my vm all of a sudden taking a crap. Once I get everything back up, ill report back here if there is still an issue

**Reply by DomJeal** [reply]
Well at least the silver lining is that these scripts make a fresh start much easier! Good luck :)

**Reply by Dvalin21** [reply]
Yeah, Im getting the same error. I tried the script (thanks tteck) and docker. UPDATE!!! Went back to version 2.10.3 and is working perfectly!!! Hope this helps someone.

**Reply by DomJeal** [reply]
Nice! Glad to hear you got it working, so weird that I'm able to run the latest version (2.10.4) without this issue then. I'm gunna treat it very nicely and hope for the best!
